generate-sdks.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+import * as colors from "colors";
+import * as fs from "fs";
+import * as path from "path";
+import * as minimist from "minimist";
+
+const repositoryName = "azure-rest-api-specs";
+const specificationsSegment = "specification";
+
+const contains = function (array, el) {
+    return array.indexOf(el) != -1
+}
+
+const args = minimist(process.argv.slice(2), {
+    boolean: ["debug", "verbose"],
+    alias: {
+        d: "debug",
+        v: "version"
+    }
+})
+
+export function findAzureRestApiSpecsRepository(): string {
+    let currentDirectory = __dirname;
+    const pathData = path.parse(currentDirectory);
```

Comment:
Please add type definitions in this file. It helps me to understand what types things are.